Sources/Common/Background Queue/QueueRunRequest.swift

```diff
@@ -67,14 +75,10 @@ public class CioQueueRunRequest: QueueRunRequest {
             logger.error("Tried to get queue task with storage id: \(nextTaskStorageId), but storage couldn't find it.")
 
             // The task failed to execute like a HTTP failure. Update `lastFailedTask`.
-            return goToNextTask(
-                queueInventory: queueInventory,
-                queryTotalNumberTasks: queryTotalNumberTasks,
-                lastFailedTask: nextTaskToRunInventoryItem
-            )
+            return goToNextTask(didTaskFail: true)
         }
 
-        logger.debug("queue tasks left to run: \(queueInventory.count) out of \(queryTotalNumberTasks)")
+        logger.debug("queue tasks left to run: \(queueInventory.count)")
```

Comment:
I think this change answers my query [here](https://github.com/customerio/customerio-ios/pull/220/files#r1033444324)